caesars-palace/blocks/carousel/carousel.js

```diff
@@ -489,4 +783,11 @@ export default async function decorate(block) {
       startAutoScroll(block, blockConfig.interval);
     }
   });
+
+  window.addEventListener('resize', () => {
+    setTimeout(() => {
+      // scroll to first slide once all DOM has been rebuilt
+      scrollToSlide(block, firstVisibleSlide, 'instant');
+    }, 0);
+  });
```

Comment:
You may want to debounce this. The `resize` event first A LOT when dragging the browser window.

```suggestion
    let resizeTimeout;
    window.addEventListener('resize', () => {
    clearTimeout(resizeTimeout);
    resizeTimeout = setTimeout(() => {
      // scroll to first slide once all DOM has been rebuilt
      scrollToSlide(block, firstVisibleSlide, 'instant');
    }, 500);
  });
```